microsoft-edge/web-platform/os-regional-settings.md

```diff
@@ -0,0 +1,85 @@
+---
+title: Updates to Region information exposed to web content
+description: How customers and web developers can use the OS regional format in Microsoft Edge for improved site experiences
+author: MSEdgeTeam
+ms.author: msedgedevrel
+ms.topic: conceptual
+ms.prod: microsoft-edge
+ms.date: 08/11/2022
+--- 
+# Updates to Region information exposed to web content
+
+## Abstract 
+Microsoft Edge provides OS regional preference information to web content in order to help web site authors curate regional tailored experiences for their customers.
+
+## Data Presentation
+Users want to see dates, times, numbers, and other formatting printed according to their preferences. For example, the US geographic region translates **7/2/2022** into **July 2nd, 2022**.  Another geographic region could translate that same date into **February 7, 2022** because they read the month and day inversely. 
+
+This feature enables website authors to deliver value to these customers in particular—customers who specifically change their regional preferences in the OS to reflect their personal needs or interests. 
+
+## Regional Codes
+The user’s formatting preferences are abstracted into a relatively simple regional code, which is used by web applications to infer the various details noted above about dates, times, and other formatting preferences. With a regional code, websites can use a built-in Internationalization API to properly format information into the user's preferences. The built-in Internationalization API saves lots of MB of data transfer over the network, keeping websites small and allowing them to quickly and easily adapt to different regions. In general, the website's internationalization customer experiences are only as good as the regional code that they're able to determine for the current user. 
+
+A regional code is a hyphenated two letter language code followed by a two-letter region code: [chromium uses ICU an implementation of the Unicode Standard’s Internationalization API](https://icu.unicode.org/home).
```

Comment:
```suggestion
A regional code is a two-letter language code followed by a hyphen and a two-letter region code.  The Chromium browser engine uses ICU, which is an implementation of the Unicode standard’s Internationalization API.  See [ICU](https://icu.unicode.org/home).
```

Reply:
@travisleithead - do we need to specify "used by the Chromium engine" here, or is it sufficient to say Microsoft Edge?  ppl are more likely to know what edge means